src/css/foundation-icons/.fontcustom-data

```diff
@@ -0,0 +1,298 @@
+{
+  "fonts": [
+    "fontcustom_10588e4d0af9f9cd5471c9540658204b.ttf",
+    "fontcustom_10588e4d0af9f9cd5471c9540658204b.woff",
+    "fontcustom_10588e4d0af9f9cd5471c9540658204b.eot",
+    "fontcustom_10588e4d0af9f9cd5471c9540658204b.svg"
+  ],
+  "templates": [
+    "fontcustom.css",
+    "fontcustom-preview.html"
+  ],
+  "file_name": "fontcustom_10588e4d0af9f9cd5471c9540658204b",
+  "glyphs": [
+    "fi-address-book",
+    "fi-alert",
+    "fi-align-center",
+    "fi-align-justify",
+    "fi-align-left",
+    "fi-align-right",
+    "fi-anchor",
+    "fi-annotate",
+    "fi-archive",
+    "fi-arrow-down",
+    "fi-arrow-left",
+    "fi-arrow-right",
+    "fi-arrow-up",
+    "fi-arrows-compress",
+    "fi-arrows-expand",
+    "fi-arrows-in",
+    "fi-arrows-out",
+    "fi-asl",
+    "fi-asterisk",
+    "fi-at-sign",
+    "fi-background-color",
+    "fi-battery-empty",
+    "fi-battery-full",
+    "fi-battery-half",
+    "fi-bitcoin-circle",
+    "fi-bitcoin",
```

Comment:
What on earth? : P

Reply:
well, the foundation icons must have a bitcoin icon.